bin/configInitializer.js

```diff
@@ -0,0 +1,131 @@
+            message: "What style of indentation do you use?",
+            default: "tabs",
+            choices: [{name: "Tabs", value: "tabs"}, {name: "Spaces", value: 4}]
+        },
+        {
+            type: "list",
+            name: "quotes",
+            message: "What quotes do you use for strings?",
+            default: "double",
+            choices: [{name: "Double", value: "double"}, {name: "Single", value: "single"}]
+        },
+        {
+            type: "list",
+            name: "linebreak",
+            message: "What line endings do you use?",
+            default: "unix",
+            choices: [{name: "Unix", value: "unix"}, {name: "Windows", value: "windows"}]
+        },
+        {
+            type: "confirm",
+            name: "semi",
+            message: "Do you require semicolons?",
+            default: true
+        },
+        {
+            type: "confirm",
+            name: "es6",
+            message: "Are you using ECMAScript 6 features?",
+            default: false
+        },
+        {
+            type: "checkbox",
+            name: "env",
+            message: "Where will your code run?",
+            default: ["browser"],
+            choices: [{name: "Node", value: "node"}, {name: "Browser", value: "browser"}]
+        },
+        {
+            type: "confirm",
+            name: "jsx",
+            message: "Will you use JSX?",
+            default: false
+        },
+        {
+            type: "confirm",
+            name: "react",
+            message: "Will you use React",
+            default: false,
+            when: function (answers) {
+                return answers.jsx;
+            }
+        },
+        {
+            type: "list",
+            name: "format",
+            message: "What format do you want your config file to be in?",
+            default: "JSON",
+            choices: ["JSON", "YAML"]
+        }
+    ], function(answers) {
+        processConfigFn(answers, createConfigFileFn, callback);
+    });
+}
+
+/* istanbul ignore next: hard to test fs function */
+/**
+ * Create .eslintrc file in the current working directory
+ * @param {object} config object that contains user's answers
+ * @param {bool} isJson should config file be json or yaml
+ * @param {function} callback callback function when finished writing .eslintrc file
+ * @returns {void}
+ */
+function writeFile(config, isJson, callback) {
+    fs.writeFile("./.eslintrc", isJson ? JSON.stringify(config, null, 4) : yaml.safeDump(config), callback);
+}
+
+var init = {
+    /**
+     * process user's answers and create config object
+     * @param {object} answers answers received from inquirer
+     * @param {function} createConfigFile function to write config file to disk
+     * @param {function} callback function to be called when file is written
+     * @returns {void}
+     */
+    processAnswers: function(answers, createConfigFile, callback) {
+        var config = {rules: {}, env: {}};
```

Comment:
I'd just have this function return an object with the processed values. Much easier to test.